### DISCUSSION TITLE
How to use this API

### DISCUSSION THREAD

**Discussion by liamgwallace** [coddingtonbear/obsidian-local-rest-api#116] [Q&A]
I am sorry if this is a dumb question, but how do I use this API? Does it interact with my browser app, does it run standalone and access the files/folders?

Does anyone have any examples please?

**Comment by flammt** [comment]
You can look at this page: https://coddingtonbear.github.io/obsidian-local-rest-api/

**Comment by CPME** [comment]
I am probably dumb, but I do nut understand how to use this page to set up and test a connection. I would appreciate a quick start guide including basic instructions for how to create a connection and a test script.
